bokeh-app/main.py

```diff
@@ -103,61 +103,68 @@ def get_data(url):
     ds = xr.open_dataset(nc_url)
     ds = ds.sel(nv=0)
     df = ds.to_dataframe()
+
+    # Check whether to fetch Sea Ice Extent or Sea Ice Area.
+    try:
+        df["sie"]
+        metric = "sie"
+    except:
+        try:
+            df["sia"]
+            metric = "sia"
+        except:
+            print("Could not find a SIE or SIA column in the data!")
+
     try:
         new_data = {
-            str(i): df["sie"].loc[df.index.groupby(df.index.year)[i]].values
+            str(i): df[metric].loc[df.index.groupby(df.index.year)[i]].values
             for i in df.index.groupby(df.index.year)
         }
     except AttributeError:
         print('multi index data using level 0')
         new_data = {
-        str(i): df["sie"].loc[df.index.groupby(df.index.get_level_values(1).year)[i]].values  
+        str(i): df[metric].loc[df.index.groupby(df.index.get_level_values(1).year)[i]].values
         for i in df.index.get_level_values(1).groupby(df.index.get_level_values(1).year)
     }
     all_years = pd.DataFrame(dict([(k, pd.Series(v)) for k, v in new_data.items()]))
     all_years.dataset_metadata = ""
     all_years.dataset_metadata = ds.attrs
     all_years.dataset_metadata["dimension"] = list(ds.dims)
     all_years.variable_metadata = ""
-    all_years.variable_metadata = ds["sie"].attrs
+    all_years.variable_metadata = ds[metric].attrs
     return all_years
 
 
-# urls = ['https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/sh/osisaf_sh_sie_daily.nc', 
+# urls = ['https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/sh/osisaf_sh_sie_daily.nc',
 #         'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/nh/osisaf_nh_sie_daily.nc',
 #         'https://hyrax.epinux.com/opendap/local_data/osisaf_nh_iceextent_daily.nc']
 
 
-urls_dict = {'South':'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/sh/osisaf_sh_sie_daily.nc', 
-        'North':'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/nh/osisaf_nh_sie_daily.nc'}
-
-df = get_data(urls_dict['North'])
-
-years = pn.widgets.MultiChoice(
-    name="Years:", options=list(df.columns), margin=(0, 20, 0, 0)
-)
-
+urls_dict = {"Sea Ice Extent":{'North':'https://thredds.met.no/thredds/dodsC/osisaf/met.no/ice/index/v2p1/nh/osisaf_nh_sie_daily.nc',
```

Comment:
This is fine for now, but in the future (10-20 regions) we will need a better way to fill this urls_dict(), e.g. from a list of supported regions.

Reply:
I am good with this PR, but it would be great if you could give it a look as well @epifanio :-)

Reply:
looks good to me as well -- regarding the selection of different regions - if each region refers to a different NetCDF resource - then this "same" code will work just fine with a few modifications.